src/components/videoPage/VideoHeaderWithDescription.tsx

```diff
@@ -63,6 +73,22 @@ export const VideoHeaderWithDescription = ({ video }: Props) => {
       <div className={s.descriptionAndButtons}>
         <div className={s.scrollableDescription}>
           <VideoDescription video={video} />
+
+          <FeatureGate feature="BO_WEB_APP_DEV">
+            {!isLoading && learningOutcomes && (
+              <div className={s.learningOutcomesWrapper}>
+                <Typography.Body weight="medium">
+                  Learning Outcomes:
+                </Typography.Body>
+                <br />
+                <div
+                  dangerouslySetInnerHTML={{
+                    __html: convertStringToHTML(learningOutcomes),
```

Comment:
I assume your other PR is related to being able to get rid of this in favour of iterating and rendering a number of list elements 👍